### ISSUE TITLE
PDF Portfolios - Not Stitching Correctly

### ISSUE THREAD

**Issue by m-prodan** [bcgov/foi-flow#3984]
**Describe the bug in current situation**
An issue was raised in production, about users being unable to see certain PDF records. When the records are loaded, and the download for harms is complete, they see this page where the PDF should be: 

![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/929d5114-3726-4836-acbe-8f12fb0f06ff)

**Link bug to the User Story**

**Impact of this bug**
Describe the impact, i.e. what the impact is, and number of users impacted. 

**Chance of Occurring (high/medium/low/very low)**
Unclear - but PDF portfolios are a somewhat uncommon, but not rare file type we see. 

Intake team reports these kinds of PDFs are common and that they are handled by Intella.

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**

**Steps to Reproduce**
1. Create a PDF portfolio
2. Upload to gathering records log
3. Download for harms

**Actual/ observed behaviour/ results**

**Expected behaviour**
With the pre-state, the Intella process was able to handle these file types. Usually what would happen is a summary page of the portfolio is created, followed by the PDFs listed. The summary page would then be deleted. 

A clear and concise description of what you expected to happen. Use the gherking language. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Pre-Deployment Steps**
- add DEDUPE_REQUEST_MANAGEMENT_API to dedupe and dedupe large files env

**Comment by lmullane** [comment]
Fix does need a parent and child relationship between the PDFs.

**Comment by m-prodan** [comment]
@nkan-aot - this appears to be working as expected; however, have noticed a few issues, and I think I've encountered a bug:

1) It appears that when it strips out the files, it adds a bunch of extraneous characters into the filename, which can make it difficult to make out. Is this fixable?

2) Oddly enough, the one portfolio I put together out of a bunch of emails threw a weird error. When I downloaded for harms, it included everything except for 2 files. When I try to download the files individually, there is no problem with the files themselves, they are clear, readable PDFs - they are just being flagged as errors during the stitch. See in TEST - JER-2022-00087, the two files in question are the bottom two files: &lt;2&gt;Re_ [bcgov_foi-flow] &#x27;Replace Manually&#x27; - new f....pdf 

3) I was working with the same set of records on LBR-2023-221234, and while deleting some files to reupload, the records log suddenly disappeared and I get: 

![image.png](https://images.zenhubusercontent.com/6081a4c82f0c363dddb9c1a0/3bb37224-a304-4411-994c-374b944485ac)

I can no longer upload files - I am not sure what happened here. Going to put this back to review. I can update you more in the morning/at stand up.
